### DISCUSSION TITLE
AOI extraction from two Images

### DISCUSSION THREAD

**Discussion by dalalprohelika** [insarlab/MintPy#1060] [General]
If my AOI is covering two images, then how should I process them together?

I am using the ASF HyP3 jupyter notebook (hyp3_insar_stack_for_ts_analysis) to process the data. Is there any direct way to process two frames together?

**Comment by welcome** [comment]
👋  Thanks for opening your first issue here! Please filled out the template with as much details as possible. We appreciate that you took the time to contribute!
Make sure you read our [contributing guidelines](https://github.com/insarlab/MintPy/blob/master/docs/CONTRIBUTING.md).

**Comment by yunjunz** [comment]
This is a duplicated question and has already been answered at https://github.com/insarlab/MintPy/issues/1027.
